crates/acap-vapix/src/lib.rs

```diff
@@ -67,8 +67,8 @@ fn from_dbus() -> anyhow::Result<HttpClient> {
 }
 
 fn from_env() -> anyhow::Result<HttpClient> {
-    let username = env::var("AXIS_DEVICE_USER")?;
-    let password = env::var("AXIS_DEVICE_PASS")?;
+    let username = env::var("AXIS_DEVICE_USER").unwrap_or("root".into());
+    let password = env::var("AXIS_DEVICE_PASS").unwrap_or("pass".into());
```

Comment:
We may need to update the checksums because of this, but let's see what the tests say.

Reply:
Seems like it wasn't needed.